### ISSUE TITLE
Chromium support

### ISSUE THREAD

**Issue by glacambre** [testimio/chrome-version#3]
Hi! :)

I see that detection of chromium is explicitly disabled on Linux  and was wondering why (I'd like to have it enabled, if there are technical limitations preventing that I'm willing to work on solving them).

https://github.com/testimio/chrome-version/blob/61d55d7df059ee26537b281c4f90d5aad6fb8b71/chrome-finder/linux.js#L52-L53 https://github.com/testimio/chrome-version/blob/61d55d7df059ee26537b281c4f90d5aad6fb8b71/chrome-finder/linux.js#L96-L100

**Comment by benjamingr** [comment]
@glacambre probably because this package is used for https://www.npmjs.com/package/chromedriver and there are no published versions for Chromium canary (only for Chrome).

I think it would be fine to add an opt-in way (with a flag) to look for it.

**Comment by glacambre** [comment]
I think I don't understand your answer, I'm currently using chromedriver 
with chromium on debian and it works just fine. In fact, the reason I 
would like chrome-version to support chromium is precisely that I would 
like chromedriver to be able to detect what chromium version I'm using 
and let it install the corresponding chromedriver binary instead of 
telling it what version I want.

I couldn't find any references to Chrome or Chromium canary in 
chromedriver's package, so I think this means chromedriver is actually 
channel-independent and only relies on the version number returned by 
`getChromeVersion`. Or am I mistaken?

Thanks for the prompt reply by the way :).

**Comment by benjamingr** [comment]
@eladrkatz

**Comment by eladrkatz** [comment]
@benjamingr If I remember correctly this is because chromium and chrome can have mismatching versions which could be problematic for us
